### ISSUE TITLE
UnicodeDecodeError on PySide 1.1.0 PKG-INFO (not UTF-8)

### ISSUE THREAD

**Issue by sjagoe** [enthought/okonomiyaki#86]
The failing sequence from the PySide PKG-INFO: ```The PySide project is developed in the open, with all facilities you\x92d expect```

```
    metadata = file_formats.EggMetadata.from_egg(path)
  File "okonomiyaki/file_formats/_egg_info.py", line 629, in from_egg
    pkg_info_data = _read_pkg_info(fp)
  File "okonomiyaki/file_formats/_package_info.py", line 245, in _read_pkg_info
    return fp.read(candidate).decode(PKG_INFO_ENCODING)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x92 in position 661: invalid start byte
```

**Comment by sjagoe** [comment]
`return fp.read(candidate).decode(PKG_INFO_ENCODING)` should handle unicode errors somehow. Possibly just like this: 
```
return fp.read(candidate).decode(PKG_INFO_ENCODING, errors='replace')
```

**Comment by sjagoe** [comment]
The file appears to be encoded as `windows-1252`, not the assumed `utf-8`.

**Comment by cournape** [comment]
Fix in #92
